src/components/review/ProposalQuestionaryReview.tsx

```diff
@@ -1,71 +1,47 @@
-import makeStyles from '@material-ui/core/styles/makeStyles';
-import Table from '@material-ui/core/Table';
-import TableBody from '@material-ui/core/TableBody';
-import TableCell from '@material-ui/core/TableCell';
-import TableRow from '@material-ui/core/TableRow';
-import Typography from '@material-ui/core/Typography';
-import React, { Fragment, HTMLAttributes } from 'react';
+import { TableProps } from '@material-ui/core';
+import React from 'react';
 
 import UOLoader from 'components/common/UOLoader';
-import QuestionaryDetails from 'components/questionary/QuestionaryDetails';
+import QuestionaryDetails, {
+  TableRowData,
+} from 'components/questionary/QuestionaryDetails';
 import { ProposalSubsetSubmission } from 'models/ProposalSubmissionState';
 
-const useStyles = makeStyles(theme => ({
-  heading: {
-    marginTop: theme.spacing(2),
-  },
-}));
-
 export default function ProposalQuestionaryReview(
-  props: HTMLAttributes<any> & {
+  props: {
     data: ProposalSubsetSubmission;
-  }
+  } & TableProps<any>
 ) {
-  const classes = useStyles();
+  const { data, ...restProps } = props;
 
-  if (!props.data.questionaryId) {
+  if (!data.questionaryId) {
     return <UOLoader style={{ marginLeft: '50%', marginTop: '100px' }} />;
   }
-  const questionary = props.data.questionary;
-  const users = props.data.users || [];
+  const questionary = data.questionary;
+  const users = data.users || [];
+
+  const additionalDetails: TableRowData[] = [
+    { label: 'Proposal ID', value: data.shortCode },
+    { label: 'Title', value: data.title },
+    { label: 'Abstract', value: data.abstract },
+    {
+      label: 'Principal Investigator',
+      value: `${data.proposer.firstname} ${data.proposer.lastname}`,
+    },
+    {
+      label: 'Co-Proposers',
+      value: users
+        .map((user: any) => ` ${user.firstname} ${user.lastname}`)
+        .toString(),
```

Comment:
I think we should use `.join(', ')` here, it would make it more consistent with other parts of the site.